src/lwc/utilCommon/utilCommon.js

```diff
@@ -36,6 +36,10 @@ const debouncify = (anyFunction, wait) => {
 const deepClone = (src) => {
```

Comment:
@bdvorachek - could we move this method below all the `is...` methods it's relying on? My IDE is complaining of usage before defining.